apphelpers/rest/hug.py

```diff
@@ -10,6 +11,20 @@
 from apphelpers.errors import InvalidSessionError
 from apphelpers.sessions import SessionDBHandler
 
+from honeybadger import Honeybadger
+
+
+# Optional Honeybadger integration. It will kick in
+# only if HONEYBADGER_API_KEY environment variable is set
+hb = None
+API_KEY = os.environ.get('HONEYBADGER_API_KEY', None)
```

Comment:
@mandarvaze we can create `APIFactory::setup_honeybadger(self, api_key):` to pass api key
cc: @shon

Reply:
Done. See updated PR.